library/sensu_check_dict.py

```diff
@@ -31,7 +31,9 @@ def main():
             name=dict(default=None, required=True),
             check_dir=dict(default='/etc/sensu/conf.d/checks', required=False),
             state=dict(default='present', required=False, choices=['present','absent']),
-            check=dict(type='dict', required=True)
+            check=dict(type='dict', required=True),
+            ucarp_node_only=dict(default=False, required=False, type='bool'),
+            vip=dict(default='', required=False)
```

Comment:
if feels like we could compress ucarp_node_only and vip into one argument_spec entry.
It would also be nice to remove 'ucarp' from our argument_spec since in general we're trying to decouple from it.
Could we have one 'only_on_ip'  entry ?  If is empty, do nothing, if it's set then use to filter the checks.